theme-release-110.rst

```diff
@@ -0,0 +1,77 @@
+.. post:: October 14, 2022
+    :tags: theme, release
+    :author: Benjamin
+    :location: Malmö, Sweden
+
+.. meta::
+    :description lang=en:
+        Information on sphinx-rtd-theme version 1.1.0
+
+
+Theme release 1.1.0
+===================
+
+We are happy to announce the release of new version of our theme, `sphinx-rtd-theme`_. In this release, we have focused on bug fixes, backwards compatibility, and making the way for future releases.
+
+.. _sphinx-rtd-theme: https://sphinx-rtd-theme.readthedocs.io/en/stable/
+
+
+Changes and new features 💄
+---------------------------
+
+Visually, we have a couple of small tweaks that most people won't notice unless we mention them here 😇 The general strategy of the theme is to avoid breaking stuff, and that also goes for the visual parts.
+
+* The ``<kbd>`` tag now has a nicer styling.
+* Breadcrumb styling has been updated
+* Table cells with multiple paragraphs now have a correct formatting.
+* Definition lists rendered correctly in API docs
+* Issues with citation rendering and styling fixed
+* Long URLs break into several lines instead of overflowing
+
+.. image:: img/sphinx-rtd-theme-110-screenshot1.png
+   :width: 30%
+   :target: _images/sphinx-rtd-theme-110-screenshot1.png
```

Comment:
This breaks in my testing -- is there a reason we have the `target` along with the path?

Reply:
I wasn't sure what to do here, it works in the local build, whereas using `img/...` doesn't.

Reply:
Fixed with a slightly hack-ish replacement of `width` with `scale`